README.md

```diff
@@ -36,6 +36,8 @@ The table below lists the available parameters and their meaning.
 | `pump_height` | Height of the block representing the pump. |
 | `pump_length` | Length of the block representing the pump. |
 | `pump_width` | Width of the block representing the pump. |
+| `left_arm` | Robot arm name used for left arm. (ur3, ur3e, ur5, ur5e, ur10, ur10e, ur16e)|
```

Comment:
Maybe add a short comment in the like of: _Robot arm model used for the left arm. Must be one of: (ur3, ur3e, ..., ur16e)_?